pcsx2/gui/AppMain.cpp

```diff
@@ -542,11 +542,13 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack || g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
+		if (g_Conf->GSWindow.IsToggleAspectRatioSwitch && !EnableFMV && !FMVstarted && art == AspectRatio_4_3 && prevAR != art)
+			prevAR = art;
 		if (EnableFMV) {
 			DevCon.Warning("FMV on");
 			ScopedCoreThreadPause paused_core(new SysExecEvent_SaveSinglePlugin(PluginId_GS));
 			if (g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
-				AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
 				if (art != AspectRatio_4_3) {
 					prevAR = art;
 					art = AspectRatio_4_3;
```

Comment:
No, it only does that when art is different from prevAR. If the user has selected a different aspect ratio in the menu, it will no longer use the original initialization. It's to catch possible manual changes from the menu aspect ratio or switching with F6 to 4:3, preventing undesired changes from or to widescreen.